Makefile.defs

```diff
@@ -16,7 +16,11 @@ TOOLCHAIN_DIR=$(MAKEFILE_DIR)toolchain
 ANDROID_ARCH?=arm
 ANDROID_TOOLCHAIN=$(TOOLCHAIN_DIR)/android-toolchain-$(ANDROID_ARCH)
 NDK?=$(TOOLCHAIN_DIR)/android-ndk-r15c
-NDKABI?=14
+ifeq ($(ANDROID_ARCH), arm64)
```

Comment:
x86_64?

Oh boy, I miss the android emulator :)

Reply:
The newer ones don't run 32-bit x86 or what do you mean?

Reply:
Nope :/

See https://github.com/koreader/koreader/issues/9446#issuecomment-1219345550

Reply:
Holy @#$@#$.